src/main/java/bank/account/AccountController.java

```diff
@@ -0,0 +1,39 @@
+package bank.account;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/persons/{id}/accounts")
+@RequiredArgsConstructor
+public class AccountController {
+    private final AccountService accountService;
+
+    @GetMapping
+    public List<AccountDTO> findAll(@PathVariable("id") String person_id, Pageable pageable) {
```

Comment:
В Java для змінних не використовую підкреслювання 
```suggestion
    public List<AccountDTO> findAll(@PathVariable("id") String personId, Pageable pageable) {
```